cpp/src/snmg/link_analysis/pagerank.cu

```diff
@@ -202,27 +183,27 @@ gdf_error gdf_multi_pagerank(const size_t n_gpus, gdf_column *src_ptrs, gdf_colu
         auto p = omp_get_num_threads(); 
         printf("\n Excecuting omp thread %d", i);
         /*cudaPointerAttributes attr;
-        cudaPointerGetAttributes (&attr, src_ptrs[i].data);
+        cudaPointerGetAttributes (&attr, src_col_ptrs[i]->data);

```

Comment:
line 179 - the print statement should be in a DEBUG block or removed before finial release

Reply:
I'll overwrite all this whole function in my next PR.
This one is a just doing a smoke test, I'll replace it with the actual PageRank.